Dockerfile

```diff
@@ -1,23 +1,17 @@
-FROM openmicroscopy/omero-ssh-c7:0.1.0
+FROM openmicroscopy/omero-ssh
```

Comment:
@joshmoore I went through first test of webspace v2 using this base with a success. Could you move omero-ssh to the new repo please, I need to add small fix, it is missing 

```
# To avoid error: sudo: sorry, you must have a tty to run sudo
RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```